.github/workflows/main.yml

```diff
@@ -46,6 +46,13 @@ jobs:
       - name: Set up Depot CLI
         uses: depot/setup-action@v1
 
+        #TODO: change before merge -- gh release download -D dep/ -R ${{ github.repository_owner }}/riscv-pk v${{ env.RISCV_PK_VERSION }} --archive=tar.gz
```

Comment:
We can get rid of this `gh` tool now that everything is public. We can also change the Makefile urls to fetch the branches directly. (after we get rid of the `feature/...`).

```
diff --git a/Makefile b/Makefile
index 34410d0..112f1d1 100644
--- a/Makefile
+++ b/Makefile
@@ -27,7 +27,7 @@ UNAME:=$(shell uname)
 TAG ?= devel
 TOOLCHAIN_REPOSITORY ?= cartesi/toolchain
 TOOLCHAIN_TAG ?= 0.15.0
-KERNEL_VERSION ?= 5.15.63-ctsi-2
+KERNEL_VERSION ?= 6.5.6-ctsi-y
 KERNEL_SRCPATH := linux-$(KERNEL_VERSION).tar.gz
 RISCV_PK_VERSION ?= 1.0.0-ctsi-1
 RISCV_PK_SRCPATH := riscv-pk-$(RISCV_PK_VERSION).tar.gz
@@ -120,7 +120,7 @@ cartesi-linux-config:
 	cp $(KERNEL_CONFIG) ./cartesi-linux-config
 
 $(KERNEL_SRCPATH):
-	wget -O $@ https://github.com/cartesi/linux/archive/v$(KERNEL_VERSION).tar.gz
+	wget -O $@ https://github.com/cartesi/linux/archive/refs/heads/feature/$(KERNEL_SRCPATH)
 
 $(RISCV_PK_SRCPATH):
 	wget -O $@ https://github.com/cartesi/riscv-pk/archive/v$(RISCV_PK_VERSION).tar.gz
diff --git a/configs/default-linux-config b/configs/default-linux-config
index 15e26b5..fa9040e 100644
--- a/configs/default-linux-config
+++ b/configs/default-linux-config
@@ -74,3 +74,4 @@ CONFIG_SLAB=y
 # CONFIG_ALLOW_DEV_COREDUMP is not set
 CONFIG_JUMP_LABEL=y
 # CONFIG_LEGACY_PTYS is not set
+CONFIG_RISCV_SBI_V01=y
diff --git a/shasumfile b/shasumfile
index d7d37cc..9d626be 100644
--- a/shasumfile
+++ b/shasumfile
@@ -1,2 +1,2 @@
-e8d4d1882632eac7fd8e433b4eb4db014fd56e645fdad3b89be7d1ca4f20ca07  linux-5.15.63-ctsi-2.tar.gz
+c765b7f7b5cd30d812c7e2adec648138646b3d05a1a335b59a9b676eafe011bc  linux-6.5.6-ctsi-y.tar.gz
 9a873345b9914940e7bf03a167da823910c8a2acadd818b780ffbd1a3edcc4c5  riscv-pk-1.0.0-ctsi-1.tar.gz
```